tactics/tactics.ml

```diff
@@ -3835,7 +3834,8 @@ let specialize_eqs id =
 	| _ ->
 	    if in_eqs then acc, in_eqs, ctx, ty
 	    else
-	      let e = e_new_evar (push_rel_context ctx env) evars t in
+              let sigma, e = Evarutil.new_evar (push_rel_context ctx env) !evars t in
+              evars := sigma;
```

Comment:
Even if somehow arbitrary and anyway imperfect, my esthetics on this code that I wrote would be pleased here and at other place of `cases.ml` if the extra `let sigma = new_evar !evars in evars := sigma` business was hidden behind a  version of e_new_evar local to `cases.ml`.